tests/integration-tests/tests/configure/test_pcluster_configure.py

```diff
@@ -74,9 +131,26 @@ def get_unsupported_test_runner_options(request):
 
 
 def orchestrate_pcluster_configure(
-    region, key_name, scheduler, os, instance, vpc_id, public_subnet_id, private_subnet_id, config_path
+    region,
+    key_name,
+    scheduler,
+    os,
+    instance,
+    vpc_id,
+    public_subnet_id,
+    private_subnet_id,
+    config_path,
+    vpc_stack,
```

Comment:
`vpc_stack` is only used to get `vpc_stack_public_subnet_id`, but we are passing in `public_subnet_id` already, can we just use that?

Reply:
I see. Then let's pass in `vpc_stack.cfn_outputs["PublicSubnetId"]` as a parameter and not pass in the entire `vpc_stack`.

Minor: Alternatively we can name things like `headnode_subnet_response` to make clear that the subnet_id is only used for response.

Please avoid using `master` in any new code that we write